### ISSUE TITLE
main: Error compiling example.tex in pdflatex

### ISSUE THREAD

**Issue by tusooa** [zepinglee/citeproc-lua#32]
When using the version on `main`, `pdflatex example.tex` gives out the following error:

```
> \g__csl_bibliography_setup_tl=\cslsetup { class = in-text, hanging-indent =
true, line-spacing = 2, entry-spacing = 0, }.
<recently read> }

l.19 \begin{document}

?
```

Using lualatex does not give out this error.

**Comment by zepinglee** [comment]
Can you give a full example like in <https://github.com/zepinglee/citeproc-lua/issues/28#issue-1452287136>?

**Comment by tusooa** [comment]
> Can you give a full example like in [#28 (comment)](https://github.com/zepinglee/citeproc-lua/issues/28#issue-1452287136)?

It's the example.tex in the repository. https://github.com/zepinglee/citeproc-lua/blob/main/example/example.tex

**Comment by zepinglee** [comment]
Ah yes. I can reproduce this bug.

**Comment by zepinglee** [comment]
https://github.com/zepinglee/citeproc-lua/blob/21f336d9dce169abd0f106ddf7f9a6700592e188/latex/citation-style-language.sty#L813

This line (equivalent to `\show`) blocks the running of latex and it is fixed in d2e3359.

https://github.com/zepinglee/citeproc-lua/blob/d2e3359f712255da278b1d81a83dbd60b796c537/latex/citation-style-language.sty#L822

**Comment by zepinglee** [comment]
I also add running of the example in GitHub Action in [31b1af7](https://github.com/zepinglee/citeproc-lua/commit/31b1af71fca334710b43870ad2c4b1aae6e2e64a).
